yellow.sdef

```diff
@@ -20,6 +20,8 @@ buildVars:
 #endif
 }
 
+#include "sinc/octave_full.sinc"
```

Comment:
I'd prefer to have a `yellow_octave.sdef` that includes `yellow.sdef` and `octave_full.sinc`; especially since people outside Sierra don't have access to the Octave .app files yet.
Of course, then we'll also need to update the Makefile to provide a `yellow_octave` goal (which should be used when building the factory spk).

Reply:
This turned out to be a bit of a pain, so I am proposing the environment variable based `OCTAVE=1` or `OCTAVE=0` to say whether octave should be included in the yellow build.